src/main.py

```diff
@@ -202,7 +326,7 @@ def _gen_cmd_add_homework():
 # Homework command
 def _cmd_homework(_tb, _message, _args):
     now = datetime.datetime.now().timestamp()
-    hw_list = database.get_hw_since(now - 60 * 60 * 24)
+    hw_list = database.get_hw_since(now - 60 * 60 * 24 * 7)
```

Comment:
now homework will be deleted only after the week passes from homework deadline

Reply:
It's not ever **deleted**... It's not **shown**